File: Wallet/Wallet.Api/DataAccess/Transaction.cs
using System;
using System.Collections.Generic;

namespace Wallet.Api.DataAccess
{
    public partial class Transaction
    {
        public Guid TransactionId { get; set; }
        public int FromUserId { get; set; }
        public int ToUserId { get; set; }
        public decimal Amount { get; set; }
        public string Purpose { get; set; }
        public DateTime TransactionAt { get; set; }
        public bool IsSuspicious { get; set; }

        public virtual User FromUser { get; set; }
        public virtual User ToUser { get; set; }
    }
}

File: wallet-ui/src/app/services/api/models/transaction.ts
import { User } from './user';

export interface Transaction {
  transactionId?: string;
  fromUserId?: number;
  toUserId?: number;
  amount?: number;
  purpose?: string;
  transactionAt?: string;
  isSuspicious?: boolean;

  fromUser?: User;
  toUser?: User;
}
